app/src/main/java/org/greenstand/android/TreeTracker/models/SessionUploader.kt

```diff
@@ -24,8 +24,7 @@ class SessionUploader(
                 targetWallet = session.destinationWallet,
                 organization = session.organization ?: "",
                 deviceConfigId = dao.getDeviceConfigById(session.deviceConfigId!!)!!.uuid,
-                startTime = session.startTime
-            )
+                startTime = session.startTime.toString().toLong()            )
```

Comment:
Please keep formatting in mind and try to follow how the rest of the code is formatting.